global/prometheus-alertmanager-operated/templates/am-config-inhibit-rules-keppel.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: inhibit-rules-keppel
+  labels:
+    alertmanager: {{ include "alertmanagerRelease.name" . }}
+spec:
+  inhibitRules:
+{{- range tuple "ap-ae-1" "ap-au-1" "ap-cn-1" "ap-jp-1" "ap-jp-2" "ap-sa-1" "ap-sa-2" "eu-de-1" "eu-de-2" "eu-nl-1" "la-br-1" "na-ca-1" "na-us-1" "na-us-2" "na-us-3" "qa-de-1" }}
```

Comment:
use the withQADE1 helper

Reply:
The regions template is not a list. Will but it in such to make it reusable.